src/server.js

```diff
@@ -16,10 +16,11 @@ var connect     = require('connect'),
  */
 function Server(host, port, key, cert)
 {
-    var server      = null,
-        handlers    = [],
+    var handlers    = [],
         requests    = [],
         connections = [];
+    
+    this.server = null
```

Comment:
I'm not a big fan of fully exposing the server instance. I would suggest to just add a `getPort()` method and rollback `this.server` to `var server`.

Reply:
Done.